frontend/src/components/Dashboard.js

```diff
@@ -0,0 +1,223 @@
+      >
+        View Registered players
+      </A>
+      <A
+        href="/validate-rosters"
+        class="block w-full px-4 py-2 border-b border-gray-200 cursor-pointer hover:bg-gray-100 hover:text-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-700 focus:text-blue-700 dark:border-gray-600 dark:hover:bg-gray-600 dark:hover:text-white dark:focus:ring-gray-500 dark:focus:text-white"
+      >
+        Validate Event Roster
+      </A>
+    </div>
+  );
+};
+
+const Dashboard = () => {
+  const [store, { setLoggedIn, setData }] = useStore();
+
+  createEffect(() => {
+    if (!store.loggedIn) {
+      const navigate = useNavigate();
+      navigate("/login", { replace: true });
+    }
+  });
+
+  onMount(() => {
+    if (!store?.data?.username) {
+      fetchUserData(setLoggedIn, setData);
+    }
+  });
+
+  createEffect(() => {
+    if (store?.data?.player || store?.data?.wards) {
+      initFlowbite();
+    }
+  });
+
+  return (
+    <div>
+      <h1 class="text-4xl font-bold mb-4 text-red-500">
+        Welcome {store?.data?.full_name || store?.data?.username}!
+      </h1>
+      <div
+        id="accordion-flush"
+        data-accordion="collapse"
+        data-active-classes="bg-white dark:bg-gray-900 text-gray-900 dark:text-white"
+        data-inactive-classes="text-gray-500 dark:text-gray-400"
+      >
+        <Show when={store.data.player}>
+          <h2 id="accordion-heading-player">
+            <button
+              type="button"
+              class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+              data-accordion-target="#accordion-body-player"
+              aria-expanded="false"
+              aria-controls="accordion-body-player"
+            >
+              <span>Player Information: {store.data.player.full_name}</span>
+              <AccordionDownIcon />
+            </button>
+          </h2>
+          <div
+            id="accordion-body-player"
+            class="hidden"
+            aria-labelledby="accordion-heading-player"
+          >
+            <div class="py-5 border-b border-gray-200 dark:border-gray-700">
+              <Player player={store.data.player} />
+            </div>
+          </div>
+        </Show>
+        <Show when={store.data.wards}>
+          <For each={store.data.wards}>
+            {ward => (
+              <>
+                <h2 id={`accordion-heading-ward-${ward.id}`}>
+                  <button
+                    type="button"
+                    class="flex items-center justify-between w-full py-5 font-medium text-left text-gray-500 border-b border-gray-200 dark:border-gray-700 dark:text-gray-400"
+                    data-accordion-target={`#accordion-body-ward-${ward.id}`}
+                    aria-expanded="false"
```

Comment:
fcc861224accd464b638ea9dcc2a9825aaed68dd also expands the wards accordions "if required"

Reply:
Same as [here](https://github.com/india-ultimate/hub/pull/75#discussion_r1309637177)